lib/easypost/services/billing.rb

```diff
@@ -12,7 +14,7 @@ def self.get_payment_method_info(priority)
     payment_method_to_use = payment_method_map[priority]
 
     error_string = 'The chosen payment method is not valid. Please try again.'
```

Comment:
issue: this should be an error constant like the other messages

Reply:
Resolved